src/ProxyClient/Varnish.php

```diff
@@ -109,7 +109,7 @@ public function invalidateTags(array $tags)
      */
     public function getTagsHeaderValue(array $tags)
     {
-        return implode(',', array_unique($this->escapeTags($tags)));
+        return array_unique($this->escapeTags($tags));
```

Comment:
hm, but is comma separating defined? the way we invalidate e.g. in
varnish, it must be comma. it seems kind of strange that psr response
would handle this, as its not a question on the HTTP layer but on
application layer. (and comma is just our choice, we could have used any
character as long as we escape it when it occurs in tags)